src/Tribe/Integrations/Freemius.php

```diff
@@ -0,0 +1,386 @@
+	 */
+	public function get_plugin_icon_url() {
+		$class = $this->object_class;
+
+		return $class::instance()->plugin_url . '/src/resources/images/' . $this->slug . '.svg';
+	}
+
+	/**
+	 * Get the Settings page path.
+	 *
+	 * @since TBD
+	 *
+	 * @return string The Settings page path.
+	 */
+	public function get_settings_path() {
+		if ( class_exists( 'Tribe__Events__Main' ) ) {
+			$url = sprintf( 'edit.php?post_type=%s&page=%s', Tribe__Events__Main::POSTTYPE, $this->page );
+		} else {
+			$url = sprintf( 'admin.php?page=%s', $this->page );
+		}
+
+		return $url;
+	}
+
+	/**
+	 * Action to skip activation since Freemius code does not skip correctly here.
+	 *
+	 * @since  TBD
+	 *
+	 * @return bool Whether activation was skipped.
+	 */
+	public function action_skip_activation() {
+		$fs_action = tribe_get_request_var( 'fs_action' );
+
+		// Prevent fatal errors.
+		if ( ! function_exists( 'fs_redirect' ) || ! function_exists( 'fs_is_network_admin' ) ) {
+			return false;
+		}
+
+		// Actually do the skipping of connection, since Freemius code does not do this.
+		if ( $this->slug . '_skip_activation' !== $fs_action ) {
+			return false;
+		}
+
+		check_admin_referer( $this->slug . '_skip_activation' );
+
+		$this->instance->skip_connection( null, fs_is_network_admin() );
+
+		fs_redirect( $this->instance->get_after_activation_url( 'after_skip_url' ) );
+
+		return true;
+	}
+
+	/**
+	 * Filter the content for the Freemius Popup.
+	 *
+	 * @since  TBD
+	 *
+	 * @param string $message         The message content.
+	 * @param string $user_first_name The first name of user.
+	 * @param string $product_title   The product title.
+	 * @param string $user_login      The user_login of user.
+	 * @param string $site_link       The site URL.
+	 * @param string $freemius_link   The Freemius URL.
+	 *
+	 * @return string
+	 */
+	public function filter_connect_message_on_update(
+		$message, $user_first_name, $product_title, $user_login, $site_link, $freemius_link
+	) {
+		$class = $this->object_class;
+
+		wp_enqueue_style( 'tribe-' . $this->slug . '-freemius', $class::instance()->plugin_url . '/src/resources/css/freemius.css' );
+
+		// Add the heading HTML.
+		$plugin_name = $this->name;
+		$title       = '<h3>' . sprintf( esc_html__( 'We hope you love %1$s', 'event-tickets' ), $plugin_name ) . '</h3>';
+		$html        = '';
+
+		// Add the introduction HTML.
+		$html .= '<p>';
+		$html .= sprintf( esc_html__( 'Hi, %1$s! This is an invitation to help our %2$s community. If you opt-in, some data about your usage of %2$s will be shared with our teams (so they can work their butts off to improve). We will also share some helpful info on events management, WordPress, and our products from time to time.', 'event-tickets' ), $user_first_name, $plugin_name );
+		$html .= '</p>';
+
+		$html .= '<p>';
+		$html .= sprintf( esc_html__( 'And if you skip this, that\'s okay! %1$s will still work just fine.', 'event-tickets' ), $plugin_name );
+		$html .= '</p>';
```

Comment:
:no_entry_sign: A gettext call containing placeholders was found, but was not accompanied by a "translators:" comment on the line above to clarify the meaning of the placeholders.

<sub>hash: 3297558e038c583c9f22e495df0c24c8</sub>